declarations.d.ts

```diff
@@ -2,6 +2,7 @@ declare module 'react-spring';
 declare module '*.png';
 declare module '*.jpg';
 declare module '*.jpeg';
+declare module '*.svg?url';
```

Comment:
see [Use SVGR and asset SVG in the same project](https://react-svgr.com/docs/webpack/#use-svgr-and-asset-svg-in-the-same-project)